app/controllers/items_controller.rb

```diff
@@ -279,7 +279,11 @@ def purge_object
     ActiveFedora.solr.conn.commit
 
     respond_to do |format|
-      format.any { redirect_to '/', notice: params[:id] + ' has been purged!' }
+      if params[:bulk]
+        format.html { render plain: 'Purged.' }
+      else
+        format.any { redirect_to '/', notice: params[:id] + ' has been purged!' }
+      end
```

Comment:
this fixes a related problem that currently exists, where the user sees a bogus "not found" error message when bulk purging items

Reply:
Can you add a test for this case?

Reply:
test added to `items_controller_spec`